plugins/generator-1.16.5/forge-1.16.5/templates/enchantment.java.ftl

```diff
@@ -98,6 +98,14 @@ public class ${name}Enchantment extends ${JavaModName}Elements.ModElement{
 			return ${data.isAllowedOnBooks};
 		}
 
+		@Override public boolean canGenerateInLoot() {

```

Comment:
How does this parameter differ from "is treasure enchantment"?

Reply:
This parameter, if set to false, prevent the enchantment to be in loot tables, not just the one of treasures.